rpcd/playbooks/roles/rpc_maas/defaults/main.yml

```diff
@@ -448,3 +448,10 @@ maas_excluded_checks: []
 # This is necessary until LP #1537117 is implemented
 openrc_os_endpoint_type: internalURL
 openrc_insecure: "{{ (keystone_service_adminuri_insecure | bool or keystone_service_internaluri_insecure | bool) | default(false) }}"
+# Name of the virtual env to deploy into (maas)
+maas_venv_tag: "{{ rpc_release }}"
+maas_venv: "/openstack/venvs/maas-{{ maas_venv_tag }}/"
```

Comment:
the `/` at the end isn't needed, and confuses things down below, it's better to leave it off the end and add it when needed (like on the next line)